spooky-oysters/bangazon-web-app
# Ticket #2 - close order

## Link to Ticket
Closes #2 

## Description of Proposed Changes

- Added an order controller and view models to support closing an order out

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout km-close-order

```
1. Build project
1. log in as a user - UNO@UNO.COM for example
1. Add ```/Order``` to the end of the URL
1. You should see a list of items with an order quantity next to each one - no order total yet
1. Click 'Complete Order'
1. You should see a drop down with payment types
1. Select a valid payment type
1. Click done
1. You should see a confirmation message
1. Clicking the link should return you to the main index page


## Impacted Areas in Application

List general components of the application that this PR will affect:

* Controller/OrderController
* Model/ViewModels - ShoppingCart view models and complete order view models
* View/Product - additional views to support completing an order

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
